iFixit/Veryable
# GitHub GraphQL validation

# Description

I found that [@octokit](https://github.com/octokit) has a repository that defines their [GraphQL schema ](https://github.com/octokit/graphql-schema) and on top of that, provide a validation tool for the GraphQL queries. 


## Changes

This integrates GitHub's GraphQL validator and schema definitions to replace the self-created definitions for the GitHub GraphQL responses https://github.com/iFixit/Veryable/blob/main/types/global.d.ts
 
This pull also adds additional tests to validate the current queries and to be used to validate future queries. 

## QA

Check everything still works and the tests pass, similar to https://github.com/iFixit/Veryable/pull/17#issue-1055586897

Closes #14
